src/Microsoft.AspNet.Identity/IIdentityKeyNormalizer.cs

```diff
@@ -4,15 +4,15 @@
 namespace Microsoft.AspNet.Identity
 {
     /// <summary>
-    ///     Used to normalize a user name
+    ///     Used to normalize keys for consistent lookups
     /// </summary>
-    public interface IUserNameNormalizer
+    public interface IIdentityKeyNormalizer
```

Comment:
Sounds good (either, I am inclined to keep Identity in the name though, assuming we do it for other interfaces).